### PR TITLE
Fix footer resizing.

### LINKED ISSUE #70: Not collapsing to a single column as it should
@jiffyclub reports that the site does not collapse to a single column on mobile as it should (see screenshots). The http://datacarpentry.org site _does_ collapse, so this is almost certainly something that @gvwilson broke during the conversion (using fixed columns for layout instead of using the template properly).
img_9518
img_9519
img_9520
img_9521

### PR DESCRIPTION
**Pull request by andreabedini**
Fixed height and buttons colliding with "services" above caused the
misaligned. Now "services" resize to a line on its own and the rest is mostly
bikeshedding.

Fix #70 again.

### CODE REVIEW

File: _sass/_07_layout.scss [hunk 1]
```diff
@@ -278,18 +277,17 @@ h3.masthead {
     padding-bottom: 0px;
     background: $footer-bg;
     color: $footer-color;
-}
```
Comment (gvwilson) [inline _sass/_07_layout.scss:281]: Why the indentation change?

**Comment by gvwilson** [comment]
Seems sensible to me, but I don't have a device I can test this on. @jiffyclub @djacquel @todrobbins can one of you please check it out?

File: _sass/_07_layout.scss (hunk 1 above)
Comment (andreabedini) [inline _sass/_07_layout.scss:281]: Wasn't intentional sorry.

**Comment by andreabedini** [comment]
To be honest I tested this just by resizing the browser window :)

**Comment by gvwilson** [comment]
I'll go ahead and merge it, and we can back it out later if need be - 
thanks!

**Pull request merged**